src/content/en/2021/structured-data.md

```diff
@@ -1,20 +1,595 @@
+
+Beyond this outlier, adoption drops off steeply. The next most common tags are `twitter:title` and `twitter:description` (both also used to define how shared URLs are presented), which appear on 20.86% and 18.68% of all pages, respectively.
+
+It's understandable why these particular tags—as well as the `twitter:image` tag (11.41% of pages) and `twitter:url` tag (3.13% of pages)—aren't more prevalent, as Twitter falls back to the equivalent Open Graph tags (`og:title`, `og:description` and `og:image`) when they're not defined.
+
+Also of interest are:
+
+* The `twitter:site` tag (11.31% of pages) which defines the Twitter account associated with the website in question.
+* The `twitter:creator` tag (3.58% of pages), which defines the Twitter account of the author of the webpage's content.
+* The `twitter:label1` and `twitter:data1` tags (both on 6.85% of pages), which can be used to define custom data and attributes about the webpage. Additional label/data pairs (e.g., `twitter:label2` and `twitter:data2`) are also present on a significant number (0.5%) of pages.
+
+Beyond these examples, usage drops off rapidly, into a long tail of (often malformed, deprecated or erroneous) tags.
+
+##### Facebook
+
+In addition to Open Graph tags, Facebook supports additional metadata (meta tags, prefixed with `fb:`) for relating webpages to specific brands, properties and people on their platform.
+
+{{ figure_markup(
+  image="structured-data-facebook.jpg",
+  caption="Facebook meta tag usage",
+  description="Bar chart showing the usage of Facebook meta tags across the web. Of all mobile pages in our data: `fb:app_id` is on 6.06%, `fb:admins` is on 2.63%, `fb:pages` is on 0.86%, `fb:page_id` is on 0.13%, `fb:profile_id` is on 0.06%, `fb:use_automatic_ad_placement` is on 0.01%, and `fb:article_style` is on less than 0.01%. Desktop usage is similar.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3kZ1Ys-9tId-WBa_8muMzrTAu1Ad5TXYgkopXMmBVc1xmd2N_4PZIpEZEOqRL7baymle0kHzaC6KY/pubchart?oid=1536454005&format=interactive",
+  sheets_gid="1437193020",
+  sql_file="facebook_types.sql"
+  )
+}}
+
+Of all of the Facebook tags that we detected, there are only three tags with significant adoption.
+
+Those are `fb:app_id`, `fb:admins`, and `fb:pages`; which we found on 6.06%, 2.63% and 0.86% of pages respectively.
+
+These tags are used to explicitly relate a webpage to a Facebook Page/Brand, or to grant permissions to a user (or users)  who administrates those profiles.
+
+Anecdotally, it's unclear how well these are supported by Facebook. The platform has gone through radical changes over the past few years, and their technical documentation hasn't been well-maintained. However, many content management systems, templates and best practice guides—as well as some of Facebook's debugging tools—still include and make reference to them.
+
+##### Microformats (and microformats2)
```

Comment:
Might wanna consider dropping the brackets:

```suggestion
##### Microformats and microformats2
```

As looks a bit odd in the Table of Contents at the moment:


![image](https://user-images.githubusercontent.com/10931297/141343508-c401008c-f513-4ee5-a2cb-e207258f8098.png)